apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -627,12 +637,11 @@ async fn router(
         req.uri().path(),
     );
 
-    let route_prefix = PathBuf::from("/");
-    let route = PathBuf::from(req.uri().path())
-        .strip_prefix(route_prefix)
-        .unwrap()
-        .to_path_buf()
-        .clone();
+    let original_path = PathBuf::from(req.uri().path());
+    let route = original_path
+        .strip_prefix(PathBuf::from(path_prefix))
```

Comment:
do we have to validate whether the prefix ends with `/`

Reply:
Updated, lmk if there's anything else.